<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>person</class>
 <widget class="QWidget" name="person">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>签到次数：</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>411</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/loginback.jpg);</string>
   </property>
   <zorder>calendarWidget</zorder>
  </widget>
  <zorder>widget</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>calendarWidget</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
